include/spdlog/details/os.h

```diff
@@ -280,7 +280,7 @@ inline int utc_minutes_offset(const std::tm &tm = details::os::localtime())
     return offset;
 #else
 
-#if defined(sun) || defined(__sun) || defined(_AIX) || defined(__ORBIS__)
+#if defined(sun) || defined(__sun) || defined(_AIX) || defined(__ORBIS__) || !(defined(HAS_TM_GMTOFF) || defined(__USE_MISC))
```

Comment:
This is probably fine but if we wanted to target Tizen more specifically could use `__native_client__ `

Reply:
Is __native_client__ a reliable way to detect Tizen environment?
(I googled/DDGed it but could not way anything that selects Tizen from others)
If so, I can change this, to be a bit more precise.

I was actually looking for a way to make this field detection more OS-agnostic and possibly even suggest a PR fix upstream (spdlog has evolved a bit since the version 1.3.1 we use here).

BTW, the SFINAE tricks _do_ let me check if the tm_gmtoff field exists or not... It's just I cannot conditionally access it based on the result :\

Reply:
OK, found this: https://developer.chrome.com/native-client/reference/pnacl-c-cpp-language-support

Fixing.

Reply:
👍